examples/forms.amp.html

```diff
@@ -116,6 +117,41 @@
 </head>
 <body>
 
+<amp-analytics>
+    <script type="application/json">
+        {
+            "requests": {
+                "test-ping": "https://my-analytics.com/ping?title=${title}&acct=${account}",
+                "event": "https://my-analytics.com/ping?eid=${eventId}&elab=${eventLabel}"
+            },
+            "vars": {
```

Comment:
Nothing is wrong with this exampe but this is not necessary to showcase analytics abilities in forms. Maybe remove? Also remove ${account} from the request?